src/Worker/CronJobs.php

```diff
@@ -39,9 +36,9 @@ public static function execute($command = '')
 		logger("Starting cronjob " . $command, LOGGER_DEBUG);
 
 		// Call possible post update functions
-		// see include/post_update.php for more details
+		// see src/Database/PostUpdate.php for more details
```

Comment:
There isn't much more detail about how to use post updates!

It makes me think I should have used them for the password and plugin updates. 😅

Reply:
The post updates are for stuff than doesn't block the working with the system.

Reply:
Nevermind them, I'm specializing myself in blocking the system.